### PR TITLE
Add Cond Access info

### PR DESCRIPTION
**Pull request by get-itips**
closes https://github.com/MicrosoftDocs/OfficeDocs-SharePoint/issues/3849

### CODE REVIEW

File: SharePoint/SharePointOnline/allow-syncing-only-on-specific-domains.md [hunk 1]
```diff
@@ -52,7 +52,7 @@ To make sure that users sync OneDrive files only on managed computers, you can c
    > Make sure to add the domain GUID of the computer domain membership. If users are in a separate domain, only the domain GUID that the computer account is joined to is required.
 
    > [!IMPORTANT]
-   > This setting is only applicable to Active Directory domains. It does not apply to Azure AD domains. If you have devices which are only Azure AD joined, consider using a [Conditional Access Policy](/azure/active-directory/conditional-access/overview) instead.
+   > This setting is only applicable to Active Directory domains. It does not apply to Azure AD domains. If you have devices which are only Azure AD joined, consider using a [Conditional Access Policy](/azure/active-directory/conditional-access/overview) instead, see also [Enable conditional access support in the OneDrive sync app](enable-conditional-access.md).
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/allow-syncing-only-on-specific-domains.md:55]:
```suggestion
   > This setting is only applicable to Active Directory domains. It does not apply to Azure AD domains. If you have devices that are only Azure AD joined, consider using a [Conditional Access Policy](/azure/active-directory/conditional-access/overview) instead, For details, see [Enable conditional access support in the OneDrive sync app](enable-conditional-access.md).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

File: SharePoint/SharePointOnline/allow-syncing-only-on-specific-domains.md [hunk 2]
```diff
@@ -52,7 +52,7 @@ To make sure that users sync OneDrive files only on managed computers, you can c
    > Make sure to add the domain GUID of the computer domain membership. If users are in a separate domain, only the domain GUID that the computer account is joined to is required.
 
    > [!IMPORTANT]
-   > This setting is only applicable to Active Directory domains. It does not apply to Azure AD domains. If you have devices which are only Azure AD joined, consider using a [Conditional Access Policy](/azure/active-directory/conditional-access/overview) instead, see also [Enable conditional access support in the OneDrive sync app](enable-conditional-access.md).
+   > This setting is only applicable to Active Directory domains. It does not apply to Azure AD domains. If you have devices that are only Azure AD joined, consider using a [Conditional Access Policy](/azure/active-directory/conditional-access/overview) instead, For details, see [Enable conditional access support in the OneDrive sync app](enable-conditional-access.md).
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/allow-syncing-only-on-specific-domains.md:55]:
```suggestion
   > This setting is only applicable to Active Directory domains. It does not apply to Azure AD domains. If you have devices that are only Azure AD joined, consider using a [Conditional Access Policy](/azure/active-directory/conditional-access/overview) instead. For details, see [Enable conditional access support in the OneDrive sync app](enable-conditional-access.md).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review, apologies @get-itips, one more

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@MikePlumleyMSFt
PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**